sample-btcd.conf

```diff
@@ -254,3 +254,11 @@
 ; be disabled if this option is not specified.  The profile information can be
 ; accessed at http://localhost:<profileport>/debug/pprof once running.
 ; profile=6061
+
```

Comment:
What do you think about moving this before the mining section?  It's not a big deal, but I think it fits a little better just after the RPC server section since it requires RPC and is probably more likely to be used over the mining and debug bits.